devicon.json

```
@@ -1923,6 +1923,36 @@
         "color": "#092e20",
         "aliases": []
     },
+    {
+        "name": "djangorest",
+        "altnames": [
+            "Django REST framework"
+        ],
+        "tags": [
+            "framework",
+            "rest",
+            "api",
+            "python",
+            "web"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain",
+                "line",
+                "original-wordmark",
+                "plain-wordmark",
+                "line-wordmark"
+            ],
+            "font": [
+                "plain",
```

Whoops, we forgot to add the line icon as a font :)
```suggestion
                "plain",
                "line",
```